gtsam/geometry/Pose3.h

```diff
@@ -356,6 +356,9 @@ inline Matrix wedge<Pose3>(const Vector& xi) {
   return Pose3::wedge(xi(0), xi(1), xi(2), xi(3), xi(4), xi(5));
 }
 
+// Convenience typedef
+typedef std::pair<Pose3, Pose3> Pose3Pair;
```

Comment:
`using` instead of `typedef`?

Reply:
Point3Pair is also defined with ```using``` in Point3.h. I can change both Pose3Pair and Point3Pair.